database/mongohelper.js

```diff
@@ -0,0 +1,146 @@
+const mongoose = require('mongoose');
+const faker = require('faker');
+mongoose.connect('mongodb://localhost/reserva', {useNewUrlParser:true});
+
+const users = [
+  {name: "Robert", pic: `https://picsum.photos/id/1/200`},
```

Comment:
I think we're supposed to have our pictures uploaded to our own host, since Josh said it'd cause a problem if the picture generators went down. I don't think we needed to have the Amazon hosting or your alternatives set up at this point, but keep it in mind for the future.